Training1/airbnb/src/components/Experience/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import News from "./News";
+import news from "./fakeData";
+import './index.css'
+
+const Experience = () => {
+  return (
+    <div className="experience">
+      <h1>Khám phá Trải nghiệm</h1>
+      <p>
+        Các hoạt động độc đáo với các chuyên gia địa phương – tham gia trực tiếp
+        hoặc trực tuyến.
+      </p>
+      <div>
+        {news.map(({ id, title, essay, imgUrl }) => (
+          <News id={id} title={title} essay={essay} imgUrl={imgUrl}/>
```

Comment:
@doankhanh222 
Same as above comment <News item={item} is enough, it make the component has less props